src/main/java/de/tum/in/www1/artemis/service/exam/ExamImportService.java

```diff
@@ -319,7 +319,9 @@ private void addExercisesToExerciseGroup(ExerciseGroup exerciseGroupToCopy, Exer
                     originalProgrammingExercise.setTasks(new ArrayList<>(templateTasks));
 
                     prepareProgrammingExerciseForExamImport((ProgrammingExercise) exerciseToCopy);
-                    yield Optional.of(programmingExerciseImportService.importProgrammingExercise(originalProgrammingExercise, (ProgrammingExercise) exerciseToCopy, false, false));
+                    // TODO might need to be adjusted
```

Comment:
Is that TODO for now or the future? Why might it need to be adjusted?

Reply:
It is out of date for now and unintentionally left there - the last boolean value will need to be modified to support the new import option (currently it is always set to false, so the test case visibilities will not be overridden)

Due to further issues with the file import this is not implemented now. https://github.com/ls1intum/Artemis/issues/8562